src/Libraries/DynamoPython/Python.csproj

```diff
@@ -1,101 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ImportGroup Label="PropertySheets">
-    <Import Project="$(SolutionDir)Config\CS.props" />
```

Comment:
so - this project is gone? Why? That means the binary will be gone I guess? A similar problem with the patch installers as mentioned before? Were there public apis here? Is the code that was built here moved to another binary?

Reply:
oh, was this just like a project holding images?